app/views/markets/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="sub-header">
+  <h3>
+    <%= link_to "Create new Market!", new_market_path %>
+  </h3>
+</div>
+
+<table>
+  <% @markets.each do |m| %>
+    <tr>
+      <td>
+        <%= link_to "Edit", edit_market_path(m.id), method: :get %>
```

Comment:
Not sure if the `method: :get` part of this call is necessary, as `link_to` creates an anchor tag/html link, and those are always GET requests.